test.py

```diff
@@ -4,14 +4,43 @@
 sep = "|"
 
 
+def put_out(input):
```

Comment:
You might be able to simplify this test using parameterized tests: https://docs.pytest.org/en/stable/parametrize.html